src/Carousel/widget/components/ImageCarousel.tsx

```diff
@@ -0,0 +1,274 @@
+
+import * as React from "Carousel/lib/react";
+
+import ImageCarouselModelProps, { HeightUnits, OnClickEvent,
+    PageLocation, WidthUnits } from "./../../Carousel.d";
```

Comment:
Keep it on one line, it draws too much attention and it's not so important. Also, do not use default imports.